docs/static/logging.asciidoc

```diff
@@ -5,20 +5,21 @@ Logstash emits internal logs during its operation, which are placed in `LS_HOME/
 DEB/RPM). The default logging level is `INFO`. Logstash's logging framework is based on
 http://logging.apache.org/log4j/2.x/[Log4j 2 framework], and much of its functionality is exposed directly to users.
 
-When debugging problems, particularly problems with plugins, it can be helpful to increase the logging level to `DEBUG` 
-to emit more verbose messages. Previously, you could only set a log level that applied to the entire Logstash product. 
-Starting with 5.0, you can configure logging for a particular subsystem in Logstash. For example, if you are 
-debugging issues with Elasticsearch Output, you can increase log levels just for that component. This way 
+When debugging problems, particularly problems with plugins, it can be helpful to increase the logging level to `DEBUG`
+to get more verbose messages. Previously, you could only set a log level that applied to the entire Logstash product.
```

Comment:
Now that we're 3 minors into the 6.x release, I wonder if it's still relevant to mention that this feature was added in 5.0? It might still be helpful for support to have this info, but it also makes the content seem a bit dated. Might be worth initiating a conversation on the techpubs channel.

Reply:
Excellent point and good discussion in techpubs channel.  I added "Decide on best approach for version references in docs. Clean up accordingly." to my LS documentation meta issue so I won't forget it.